extensions/kusto/src/kustoServer.ts

```diff
@@ -35,7 +45,7 @@ export class KustoServer {
 			const installationComplete = Date.now();
 			let serverOptions = generateServerOptions(context.extensionContext.logUri.fsPath, path);
 			let clientOptions = getClientOptions(context);
-			this.client = new SqlOpsDataClient(Constants.serviceName, serverOptions, clientOptions); // TodoKusto: Update constant
+			this.client = new SqlOpsDataClient('kusto', Constants.serviceName, serverOptions, clientOptions);
```

Comment:
While you're making changes, it'd actually be nice to add a comment here explaining that the name here needs to match the first value in the config `kusto.trace.server` so we don't have someone change that without knowing and break that functionality.

(if you could add it to the MSSQL version too that'd be great...)